cli/src/utils.ts

```diff
@@ -0,0 +1,88 @@
+import { defaults, getMyUserInfo, isHttpError } from '@immich/sdk';
+import { readFile, writeFile } from 'node:fs/promises';
+import { join } from 'node:path';
+import yaml from 'yaml';
+
+export interface BaseOptions {
+  configDirectory: string;
+  apiKey?: string;
+  instanceUrl?: string;
+}
+
+export interface AuthDto {
+  instanceUrl: string;
+  apiKey: string;
+}
+
+export const authenticate = async (options: BaseOptions): Promise<void> => {
+  const { configDirectory: configDir, instanceUrl, apiKey } = options;
+
+  // provided in command
+  if (instanceUrl && apiKey) {
+    await connect(instanceUrl, apiKey);
```

Comment:
I believe you miss a return here, right?

Reply:
Yuuuuup.